src/index.html

```diff
@@ -134,12 +143,63 @@ <h2 class="modal__title">New Post</h2>
         <div class="modal__content">
           <img src="" alt="pic" class="modal__img" />
           <button
-            class="modal__close-btn modal__close-btn_light"
+            class="modal__close-btn modal__close-btn_preview"
             type="button"
           ></button>
           <p class="modal__caption"></p>
         </div>
       </div>
+      <div class="modal" id="delete-modal">
+        <div class="modal__content modal__content_delete">
+          <div class="modal__container modal__container_smaller">
+            <p class="modal__question">
+              Are you sure you want to delete this image?
+            </p>
+            <form class="modal__delete-form">
+              <button
+                class="modal__delete-btn modal__delete-btn_delete"
+                type="submit"
+              >
+                Delete
+              </button>
+              <button
+                class="modal__delete-btn modal__delete-btn_cancel"
+                type="button"
+              >
+                Cancel
+              </button>
+            </form>
+          </div>
+          <button
+            class="modal__close-btn modal__close-btn_delete"
+            type="button"
+          ></button>
+        </div>
+      </div>
+      <div class="modal" id="avatar-modal">
+        <div class="modal__container">
+          <div class="modal__header">
+            <h2 class="modal__title">Edit Profile Avatar</h2>
+            <button class="modal__close-btn" type="button"></button>
+          </div>
+          <form class="modal__form" name="post-form">
```

Comment:
`name="post-form"` is already in the card form in line 111, but here should be `name="avatar-form"`

* Please, add `name="avatar-form"` for the `avatar` form in `index.html`, to be able to correctly take it from `document.forms`